airbyte-integrations/connectors/source-amazon-ads/source_amazon_ads/streams/report_streams/report_streams.py

```diff
@@ -122,6 +122,8 @@ def __init__(self, config: Mapping[str, Any], profiles: List[Profile], authentic
         # Maximum retries Airbyte will attempt for fetching report data. Default is 5.
         self.report_generation_maximum_retries: int = get_typed_env("REPORT_GENERATION_MAX_RETRIES", 5)
 
+        self._report_record_types = config.get("report_record_types", [])
```

Comment:
all looks good!

just to be more consistent with input validation and transformation
can you please move this transformation after this line:

https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-amazon-ads/source_amazon_ads/source.py#L57

like this:
```
config["report_record_types"] = config.get("report_record_types", [])
```


in `reports_streams.py` you can simplify:
```
self._report_record_types = config["report_record_types"]
```

Reply:
Updated the code as per suggestion